app/assets/stylesheets/styleguide/_font-stacks.scss

```diff
@@ -32,6 +33,7 @@ $NTA-Light:
        local("Helvetica");
 }
 
+// scss-lint:disable NameFormat
```

Comment:
Are these variables being used? Shouldn't they be pulled from the frontend toolkit? Probably a question for a different PR.